themes/default/content/blog/google-cloud-7-0/index.md

```diff
@@ -0,0 +1,176 @@
+
+### Coverage and Growth
+
+The Google Cloud provider now supports 250+ functions, 800+ resources, 10,000+ input properties. It continues to grow in coverage and features while staying up-to-date with the latest upstream changes. The chart below shows the growth of the provider in the last 2 years.
+
+![google-cloud-coverage](./google-cloud-coverage.png)
+
+### New Modules
+
+Version 6.0 of the Google Cloud provider introduced support for several new modules, including:
+
+- [Alloy DB](https://www.pulumi.com/registry/packages/gcp/api-docs/alloydb/)
+- [Assured Workloads](https://www.pulumi.com/registry/packages/gcp/api-docs/assuredworkloads/)
+- [Backup Disaster Recovery](https://www.pulumi.com/registry/packages/gcp/api-docs/backupdisasterrecovery/)
+- [BeyondCorp](https://www.pulumi.com/registry/packages/gcp/api-docs/beyondcorp/)
+- [BigLake](https://www.pulumi.com/registry/packages/gcp/api-docs/biglake/)
+- [BigQuery Analytics Hub](https://www.pulumi.com/registry/packages/gcp/api-docs/bigqueryanalyticshub/)
+- [Certificate Manager](https://www.pulumi.com/registry/packages/gcp/api-docs/certificatemanager/)
+- [Cloud Deploy](https://www.pulumi.com/registry/packages/gcp/api-docs/clouddeploy/)
+- [Cloud IDS](https://www.pulumi.com/registry/packages/gcp/api-docs/cloudids/)
+- [Database Migration Service](https://www.pulumi.com/registry/packages/gcp/api-docs/databasemigrationservice/)
+- [Dataflow](https://www.pulumi.com/registry/packages/gcp/api-docs/dataflow/)
+- [Dataform](https://www.pulumi.com/registry/packages/gcp/api-docs/dataform/)
+- [DataPlex](https://www.pulumi.com/registry/packages/gcp/api-docs/dataplex/)
+- [Datastream](https://www.pulumi.com/registry/packages/gcp/api-docs/datastream/)
+- [Edge Network](https://www.pulumi.com/registry/packages/gcp/api-docs/edgenetwork/)
+- [GKE Backup](https://www.pulumi.com/registry/packages/gcp/api-docs/gkebackup/)
+- [GKE On-Prem](https://www.pulumi.com/registry/packages/gcp/api-docs/gkeonprem/)
+- [Looker](https://www.pulumi.com/registry/packages/gcp/api-docs/looker/)
+- [Network Connectivity](https://www.pulumi.com/registry/packages/gcp/api-docs/networkconnectivity/)
+- [Network Security](https://www.pulumi.com/registry/packages/gcp/api-docs/networksecurity/)
+- [Network Services](https://www.pulumi.com/registry/packages/gcp/api-docs/networkservices/)
+- [VMware Engine](https://www.pulumi.com/registry/packages/gcp/api-docs/vmwareengine/)
+- [Workstations](https://www.pulumi.com/registry/packages/gcp/api-docs/workstations/)
+
+If you haven't used these modules yet, now is a great time to try them out! Check out the [Pulumi Registry](https://www.pulumi.com/docs/intro/cloud-providers/google-cloud/) for more information on how to get started or ask [Pulumi AI](https://www.pulumi.com/ai) to create a new project for you.
+
```

Comment:
Since the previous section was all about the numerous updates made to the 6.x release, is it worth a header or some sort of transition to help readers understand that the next few sections are for the 7.0 release?